### DISCUSSION TITLE
HomeAssistant core 2023.10.1 - Errore "version `GLIBC_2.33' not found"

### DISCUSSION THREAD

**Discussion by katanza12** [indomus/forum#2661] [HUB: Home Assistant]
Ciao a tutti. 
Da quando ho installato 2023.10.1 di HA core, non riesco più ad avviarlo per questo errore: version 'GLIBC_2.33' not found.
Ho cercato un po' in rete, ma le uniche cose che ho trovato sono problemi legati al package "grpcio" (ma nella vesione 1.44, ora siamo alla 1.59) ma anche aggiornandolo non è cambiato nulla. La versione di Python è la 3.11.
Non so più che fare, questa è la riga completa dell'errore:
`ImportError: /lib/arm-linux-gnueabihf/libc.so.6: version 'GLIBC_2.33' not found (required by /srv/homeassistant/lib/python3.11/site-packages/cryptography/hazmat/bindings/_rust.abi3.so)`

Ogni aiuto o suggerimento sarà sentitamente apprezzato!!

**Comment by indomus** [comment]
> Ogni aiuto o suggerimento sarà sentitamente apprezzato!!

stoppa l'istanza Venv e creane una @Docker che punti sul filesystem della vecchia.
